### DISCUSSION TITLE
[TOOL] web migration tool problem

### DISCUSSION THREAD

**Discussion by chamorrorenzo** [azerothcore/azerothcore-wotlk#3895] [Tools]
I know I should ask there, but its been asked there with no real answer to this issue

"column cout doesnt match value count at row 1"
https://user-images.githubusercontent.com/7823599/49156061-0f6afa80-f2e2-11e8-98b0-5e58c1675666.png

my guess

step1
step2

but I cant figure how, after a few trys
any help?
thanks

**Comment by Helias** [comment]
tell me how to reproduce this and I will try to help you

**Comment by chamorrorenzo** [comment]
> 
> 
> tell me how to reproduce this and I will try to help you

thanks, here I upload the dump character file (chardump.lua) :
http://www.mediafire.com/file/9qdnbumlgx3y5b5/chardump.rar/file

so everytime I dump a character and I try to upload the file (chardump.lua) to the migration site,  I got that error.
You can try using this character dump file

if you need anything else tell me, thanks for the help

*Im using this tool for the first time, everything is setup correctly (dumping a fresh character from other server beeing lvl1, works)

**Reply by mnclaras** [reply]
Hey, did you solved the issue? I'm facing the same error. It can be solved changing all inserts/updates over the table character_spell...
My problem comes when I send a migration. The people cannot create any character because the character data seems crashed
